sass/px-datetime-range-field.scss

```diff
@@ -43,10 +48,43 @@ $inuit-enable-validation-states : true;
   align-items: center;
 }
 
+//FLEXIBLE LAYOUTS FOR THE FIELDS
 .dt-fields-justify {
-  justify-content: var(--px-datetime-range-field-justify-fields);
+  justify-content: var(--px-datetime-range-field-justify-fields, flex-start);
+}
+
+:host([full-width][show-field-titles]){
+  // fallback for IE ಠ_ಠ
```

Comment:
I have some horrible feeling that the emoji's will break some build 2 years from now. And we'll regret putting them in ;)

Reply:
alright I'll take it out